planetarium/NineChronicles
# Show popup on raking error

### Description

SSIA

### How to test

1. Paste this line `SharedModel.StageRankingInfos = null`
on first line in `UpdateCategoryAsync` method in `RankPopup.cs`.
2. After loading raking, check if clicking stage raking category button shows error popup.

### Related Links
https://app.asana.com/0/1133453747809944/1201425663841542

### Required Reviewers
@planetarium/9c-dev @Namyujeong
